src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -199,7 +202,26 @@ export default {
 				this.clientCategories), 'id');
 			return categories
 				// fiter our any empty categories and categories with 0 loans
-				.filter(channel => _get(channel, 'loans.values.length') > 0)
+				.filter(channel => {
+					return this.categoryServiceExpVersion === 'b'
+						? channel?.savedSearch?.loans?.values?.length > 0
+						: _get(channel, 'loans.values.length') > 0;
+				})
+				// map category server category structure to standard loan channel structure
+				.map(category => {
+					// return standard category
+					if (this.categoryServiceExpVersion === 'a') {
+						return category;
+					}
+					// return mapped Category Service category
+					return {
+						id: category?.loanChannelId,
+						name: category?.name ?? '',
+						description: category?.description ?? '',
+						url: category?.url ?? '',
+						loans: category?.savedSearch?.loans ?? []
```

Comment:
The loans from the new service are nested under a `savedSearch`. This normalizes them so we don't have to handle that separately in the row presentation components.

Reply:
Yeah I was going to ask about the nesting, it seemed a little confusing.

Reply:
It's definitely different but what's happening is they are automatically mapping a saved search id onto savedSearch so we've added a layer to handle mapping the filters to FLSS in the service...